jobs/email-reminder/requirements.txt

```diff
@@ -7,7 +7,7 @@ Werkzeug==1.0.1
 aniso8601==9.0.1
 asyncio-nats-client==0.11.4
 asyncio-nats-streaming==0.4.0
-attrs==20.3.0
+attrs==23.1.0
```

Comment:
update the library to make it compatible with updated legal-api and sql-versioning